insights/parsers/systemd/config.py

```diff
@@ -17,58 +18,55 @@ class SystemdDocker(Parser):
 
     The content of file ``/usr/lib/systemd/system/docker.service`` is recorded
     via INI format, ``ConfigParser`` could be used to parse the content.
+    
+    Typical output of the ``systemd_docker`` command is::
+    
+        [Service]
+        Type=notify
+        EnvironmentFile=-/etc/sysconfig/docker
+        EnvironmentFile=-/etc/sysconfig/docker-storage
+        EnvironmentFile=-/etc/sysconfig/docker-network
+        Environment=GOTRACEBACK=crash
+        ExecStart=/bin/sh -c '/usr/bin/docker-current daemon \\
+                  --authorization-plugin=rhel-push-plugin \\
+                  --exec-opt native.cgroupdriver=systemd \\
+                  $OPTIONS \\
+                  $DOCKER_STORAGE_OPTIONS \\
+                  $DOCKER_NETWORK_OPTIONS \\
+                  $ADD_REGISTRY \\
+                  $BLOCK_REGISTRY \\
+                  $INSECURE_REGISTRY \\
+                  2>&1 | /usr/bin/forward-journald -tag docker'
+        LimitNOFILE=1048576
 
     Example::
-
-        [Unit]
-        Description=Docker Application Container Engine
-        Documentation=http://docs.docker.com
-        After=network.target
-        Wants=docker-storage-setup.service
-        Requires=rhel-push-plugin.socket
-
+    >>> docker_content = '''
```

Comment:
This section should start with the line `Examples:` instead of `Example::` and all lines following should be indented one level.  You need to also do this for the other classes.  You can check that this is correct by running the command `make clean html_debug` from the `./docs` directory of your project and if there are no errors you can view the output by running the following commands:
```
$ cd docs/_build/html
$ firefox index.html
```